docs/lifestyles.md

```diff
@@ -38,16 +38,6 @@ Container.Register(Component.For<MyTransientComponent>().LifestyleTransient());
 
 :information_source: **What transients are good for:** Transient lifestyle is a good choice when you want to be in control of instance's lifetime. When you need new instance, with new state every time. Also transient components don't need to be thread safe, unless you explicitly use them in multi-threaded situations. In most applications you'll find that a large percentage of your components will end up as transient.
 
-### PerWebRequest
```

Comment:
Not a thing so I removed it from the main docs concerning lifestyles. We should be using scopes.